pygsti/objects/operation.py

```diff
@@ -1434,13 +1471,36 @@ def __init__(self, name, evotype):
             else:  # evotype in ('densitymx', 'svterm', 'cterm')
                 ptm = _gt.unitary_to_pauligate(U)
                 rep = replib.DMOpRepDense(LinearOperator.convert_to_matrix(ptm))
+        elif evotype == 'chp':
+            std_chp_ops = _itgs.standard_gatenames_chp_conversions()
+            if self.name not in std_chp_ops:
+                raise ValueError("Name '%s' not in standard CHP operations" % self.name)
+
+            native_ops = std_chp_ops[self.name]
+            nqubits = 2 if any(['c' in n for n in native_ops]) else 1
+
+            rep = replib.CHPOpRep(native_ops, nqubits)
         else:
             raise ValueError("Invalid evotype for a StaticStandardOp: %s" % evotype)
 
         LinearOperator.__init__(self, rep, evotype)
 
+    @property
+    def base(self):
```

Comment:
I don't think StaticStandardOp should have a .base attribute.  This should be in a to_dense method I think?

Reply:
It definitely shouldn't, this was just a temporary measure to get StaticStandardOp to work everywhere. Inserted a comment noting this, and hope that this will be fixed post evotype refactor.